<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_previous_page">
       <property name="text">
        <string>上一页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_next_page">
       <property name="text">
        <string>下一页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>每页显示组数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_show_group_count"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView_group"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
